sunkit_image/flct/tests/test_flct.py

```diff
@@ -0,0 +1,303 @@
+import os
+import sys
+
+import numpy as np
+import pytest
+
+import sunkit_image.data.test as data
+import sunkit_image.flct as flct
+
+# We skip this file as the extension is not built on windows.
+if sys.platform.startswith("win"):
+    pytest.skip("Tests will not run on windows", allow_module_level=True)
+
+
+# Testing the FLCT subroutines
+@pytest.fixture
+def arrays_test():
+
+    a = np.zeros((4, 4))
+    b = np.ones((4, 4))
+    c = np.zeros((4, 4))
+
+    return (a, b, c)
+
+
+def test_two_read_write(arrays_test):
+
+    file_name = "temp.dat"
+
+    flct.write_2_images(file_name, arrays_test[0], arrays_test[1])
+
+    arr, barr = flct.read_2_images(file_name)
+
+    os.remove(file_name)
+
+    assert np.allclose(arr, np.zeros((4, 4)))
```

Comment:
All these tests need some comments.  What are these tests meant to be testing?  The intent of these tests is not clear from reading the code, and so comments are required describing what these are meant to be testing.

Reply:
Yes I agree. Either comments or docstrings.